webapi/src/main/scala/org/knora/webapi/responders/admin/UsersResponderADM.scala

```diff
@@ -1761,10 +1795,13 @@ class UsersResponderADM(responderData: ResponderData) extends Responder(responde
   /**
    * Tries to retrieve a [[UserADM]] either from triplestore or cache if caching is enabled.
    * If user is not found in cache but in triplestore, then user is written to cache.
+   *
+   * @param identifier The identifier of the user (can be IRI, e-mail or username)
+   * @return a [[Option[UserADM]]]
    */
   private def getUserFromCacheOrTriplestore(
     identifier: UserIdentifierADM
-  ): Future[Option[UserADM]] = tracedFuture("admin-user-get-user-from-cache-or-triplestore") {
+  ): Future[Option[UserADM]] = // tracedFuture("admin-user-get-user-from-cache-or-triplestore") {
```

Comment:
if we start doing that (which I find good), we should remove it everywhere, and also get rid of our `kamon` dependency

Reply:
This is to be further discussed in the backend team. I leave it outcommented for the time being.